vitrivr-engine-index/src/main/kotlin/org/vitrivr/engine/index/decode/FFmpegVideoDecoder.kt

```diff
@@ -112,7 +112,7 @@ class FFmpegVideoDecoder : DecoderFactory {
 
                 /* Execute. */
                 try {
-                    var output = FrameOutput.withConsumerAlpha(consumer).disableStream(StreamType.SUBTITLE).disableStream(StreamType.DATA)
+                    var output = FrameOutput.withConsumer(consumer).disableStream(StreamType.SUBTITLE).disableStream(StreamType.DATA)
```

Comment:
Because the resulting image cannot be saved as JPEG if the alpha channel is included, i.e., a conversion is needed. And frankly, I'm not aware of any case where an alpha channel might be useful in a video.

Reply:
There are video formats that support alpha channels, the applications are admittedly niche. I do however not think that not being able to save images as JPEGs is a valid reason for this change, since we might as well ingest JPG images directly in an image pipeline which would result in the same image saving problem. A conversion step would be needed there in any case.

Reply:
Agreed. Made the necessary changes.